### PR TITLE
Reformat docstrings to be closer to conventional style

### PR DESCRIPTION
**Pull request by kamalmarhubi**
The main changes:
- move docstrings to be inside the item they document
- change ''' to """ as suggested in [PEP 257](https://www.python.org/dev/peps/pep-0257/#id15)
- remove leading spaces
- add some sentence ending punctuation

### CODE REVIEW

**Comment by kamalmarhubi** [comment]
The main thing I wanted to fix was the first item, since that's how the Python `help` function finds documentation. The rest was me getting carried away. I can undo any of them. :-)

File: botbuilder.py [hunk 1]
```diff
@@ -21,10 +21,10 @@ def __init__(self, zulip_username, zulip_api_key, subscribed_streams=[]):
         self.db = shelve.open('./db', writeback=True)
 
 
-    ''' Standardizes a list of streams in the form [{'name': stream}]
-    '''
     @property
     def streams(self):
+    """Standardizes a list of streams in the form [{'name': stream}]."""
```
Comment (zeusdeux) [inline botbuilder.py]: Could you fix the indentation of this comment please?
It won't run otherwise.

Comment (kamalmarhubi) [inline botbuilder.py]: Ah sorry! I didn't actually try to run it which was silly.

Done.

**Comment by zeusdeux** [comment]
@kamalmarhubi merged. Thanks :)

**Pull request merged**